lib/Controller/Report.php

```diff
@@ -808,75 +808,87 @@ public function savedReportGrid(Request $request, Response $response)
             /** @var \Xibo\Entity\SavedReport $savedReport */
 
             $savedReport->includeProperty('buttons');
-            
-            $schemaVersion = $savedReport->schemaVersion;
 
-            // Show only convert button for schema version 1
-			if ($schemaVersion == 1) {
+            // If a report class doesnot comply (i.e., no category or route) we get an error when trying to get the email template
+            // Dont show any button if the report is not compatible
+            $compatible = true;
+            try {
+                // Get report email template
+                $emailTemplate = $this->reportService->getReportEmailTemplate($savedReport->reportName);
 
-                $savedReport->buttons[] = [
-                    'id' => 'button_convert_report',
-                    'url' => $this->urlFor($request,'savedreport.convert.form', ['id' => $savedReport->savedReportId] ),
-                    'text' => __('Convert')
-                ];
-                continue;
+            } catch (NotFoundException $exception) {
+                $compatible = false;
             }
 
-            $savedReport->buttons[] = [
-                'id' => 'button_show_report.now',
-                'class' => 'XiboRedirectButton',
-                'url' => $this->urlFor($request,'savedreport.open', ['id' => $savedReport->savedReportId, 'name' => $savedReport->reportName] ),
-                'text' => __('Open')
-            ];
-            $savedReport->buttons[] = ['divider' => true];
+            if ($compatible == 1) {
```

Comment:
Sorry to be pedantic, but this should be `if ($compatible) {` ? because compatible is a boolean?